compiler/onecc-docker/README.md

```diff
@@ -0,0 +1,36 @@
+# onecc-docker
+
+_onecc-docker_ is designed to improve user usability.
+
+## Description
+
+For now, `one-cmds` tools only support Ubuntu 18.04 and 20.04(not officially).
+So, It is difficult for people in different environments to use our tools without using ubuntu 18.04.
+
+To overcome this limitation, we provide _onecc-docker_ that runs using a docker so that users can use `one-cmds` more widely.
+
+This tool aims at the following objectives.
+
+- Unsupported Ubuntu OS supports ONE tool
```

Comment:
Maybe `Support ONE tool regardless of Ubuntu version` ?
Q) there are `one-cmds` and here uses `ONE tool`. Are they different or same?

Reply:
These are the same.

@holawan Please unify `ONE tools` or `one-cmds` into one word. :)